utilitiesformtg.js

```diff
@@ -1,7 +1,29 @@
+//nodelist foreach polyfill
+if (window.NodeList && !NodeList.prototype.forEach) {
+  NodeList.prototype.forEach = Array.prototype.forEach;
+}
+
 function swapimage(id, front, back) {
     if (document.getElementById(id).src == front) {
-        document.getElementById(id).src=back;        
+        document.getElementById(id).src=back;
     } else {
         document.getElementById(id).src=front;
     }
-};
\ No newline at end of file
+};
+
+//bind touch event
+//TODO: check how we do this with dynamic content
+setInterval(function  () {

```

Comment:
can you please add a doc comment explaining why this needs to be done in an interval rather than just once?

Reply:
like this? If there would be an option to change the event and add it inline to the generated tags, would you have time to look at that with me?

Reply:
the html is defined here: https://github.com/dbu/wordpress-mtg/blob/master/utilitiesformtg.php#L242 if we can add some element that helps, sure!

is the interval thing "polling"? do we stop polling once the view is loaded?

Reply:
we would need a "touchstart" event on the cards or one of the elements around them (not sure which) like this: https://www.w3schools.com/jsref/event_touchstart.asp
and then we would need a script tage somewhere on the page which binds the function into a global space for example: window.__scryFallPreviewTouchEvent(event){
}
then we would have our function binding on all cards and need no interval anymore at all, we just need to be sure that the function is always loaded and at best its only loaded one time per page and always before we display the cards....